Tasks/AzurePolicyV0/task.json

```diff
@@ -3,7 +3,7 @@
     "name": "AzurePolicyCheckGate",
     "friendlyName": "Check Azure Policy compliance",
     "description": "Security and compliance assessment for Azure Policy",
-    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/deploy/azure-policy-check-gate",
+    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/deploy/azure-policy",
```

Comment:
This https://docs.microsoft.com/azure/devops/pipelines/tasks/deploy/azure-policy-check-gate url working and new url throwing 404 error. Can you check once.